src/store/reducer.ts

```diff
@@ -0,0 +1,19 @@
+import { createReducer } from '@reduxjs/toolkit';
+import { Cities } from '../const.ts';
+import { offers } from '../mocks/offers.ts';
+import { changeCity, selectOffers } from './action.ts';
+
+const initialState = {
+  city: Cities.PARIS,
+  offers: offers
+};
```

Comment:
Можно немного улучшить, чтобы сделать более удобным для масштабирования, читаемым и типизированным:

```suggestion
// Типизация состояния
type State = {
  city: City;
  offers: Offer[];
};

// Инициализация состояния с типами
const initialState: State = {
  city: Cities.PARIS,
  offers: offers
};
```